libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -1114,27 +1068,76 @@ def _make_message_chunk_from_anthropic_event(
     event: anthropic.types.RawMessageStreamEvent,
     *,
     stream_usage: bool = True,
+    coerce_content_to_string: bool = False,
```

Comment:
is this to avoid unnecessarily turning everything into a list of dicts instead of a str? IMO might be "less is more" and change everything to list of dict

Reply:
Wanted to limit the amount of adjustment for users for now - and wasn't sure how StrOutputParser would handle it.